### DISCUSSION TITLE
Error : No module named 'pandas.tslib'

### DISCUSSION THREAD

**Discussion by AvitayGT** [pydata/xarray#5095] [Q&A]
Hi everybody,

using Google Colab, I get the next error :

![image](https://user-images.githubusercontent.com/81641993/113027599-f36a1300-9192-11eb-876f-6da9af214f58.png)

Does anybody know what is wrong and how to fix this?

I am trying to follow some guide published on 2015, maybe some things have changed since then regarding the packages and the error arises due to the changes... idk. 

Sincerely,

Avitay

**Comment by keewis** [comment]
that sounds like a really old guide. `xarray` has changed a lot since then (it has even been renamed) and `pandas` released its first "stable" release, so of course this will break. I think you can either try to reproduce a environment from back then (which might be difficult) or you might need to figure out how to upgrade to a newer version. Can you share a link to that guide?

**Reply by AvitayGT** [reply]
Hi,
You are right, I uninstalled the xray package, installed xarray package and re-installed pandas and then the error was gone. :)
(the link to the guide is: [Here](https://hrishichandanpurkar.blogspot.com/2015/10/xray-brilliant-little-tool-for-climate.html) )

Could you please help me with a new error which I now get when trying to read the following '.nc' file? :

![image](https://user-images.githubusercontent.com/81641993/113141817-dbe26700-9232-11eb-8728-616fe51ce410.png)


**The error I get is:**


![image](https://user-images.githubusercontent.com/81641993/113142010-13511380-9233-11eb-899b-093aabefe8ed.png)

**How can I overcome this? How can I load this .nc file properly?**

**- Avitay**
